src/main/java/pl/touk/sputnik/processor/spotbugs/SpotBugsProcessor.java

```diff
@@ -107,15 +107,21 @@ private IClassScreener createClassScreener(@NotNull Review review) {
 
     @Nullable
     private String getIncludeFilterFilename() {
-        String includeFilterFilename = config.getProperty(GeneralOption.FINDBUGS_INCLUDE_FILTER);
-        log.info("Using FindBugs include filter file {}", includeFilterFilename);
+        String includeFilterFilename = config.getProperty(GeneralOption.SPOTBUGS_INCLUDE_FILTER);
+        if (StringUtils.isBlank(includeFilterFilename)) {
+            includeFilterFilename = config.getProperty(GeneralOption.FINDBUGS_INCLUDE_FILTER);
```

Comment:
I do not like reassigning variable values...

Reply:
Me neither but it's reasonable here, WDYT?